<!--
  Standalone launcher for the velocity smoother
 -->
<launch>
  <node pkg="nodelet" type="nodelet" name="nodelet_manager"  args="manager"/>
  <node pkg="nodelet" type="nodelet" name="velocity_smoother" args="load yocs_velocity_smoother/VelocitySmootherNodelet nodelet_manager">
    <rosparam file="$(find yocs_velocity_smoother)/param/standalone.yaml" command="load"/>
    <remap from="velocity_smoother/raw_cmd_vel" to="raw_cmd_vel"/>
    <remap from="velocity_smoother/odom" to="odom"/>
    <remap from="velocity_smoother/smooth_cmd_vel" to="cmd_vel_mux/smooth_cmd_vel"/>
  </node>
</launch>